CHANGELOG.md

```diff
@@ -1,277 +1,731 @@
-  firmware logs.
-
-## v1.0.1 - 2022-09-02
-
-* Added system76_ectool to enable Embedded Controller [firmware
-  updating](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#dasharo-ec-update).
-* Added ec_transition script which helps with full Dasharo/Embedded Controller
-  [firmware transition](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#dasharo-ec-transition)
-  for NS50 70MU and NS70 laptops.
-* First public release: <https://github.com/Dasharo/meta-dts>.
-
-## v1.0.0 2022-08-09
-
-* Initial release v1.0.0.
-* Auto-login functionality.
-* User menu.
-* [Dasharo HCL
-  Report](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#dasharo-hcl-report)
-  \- the ability to automatically dump device information and send it to 3mdeb
-  servers.
-* Possibility to manually [update the Dasharo
-  firmware](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#dasharo-firmware-update).
-* [Bootable via
-  iPXE](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#bootable-over-network).
-* [Bootable via
-  USB](https://docs.dasharo.com/common-coreboot-docs/dasharo_tools_suite/#bootable-usb-stick).
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [2.0.0-rc4] - 2024-07-26
+
+### Added
+
+- distro: recipes-extended: minio-cli: add
+- distro: recipes-dts: dts-scripts: add DES packaging support
+
+### Changed
+
+- dts-scripts: update SRCREV
+
+## [2.0.0-rc3] - 2024-07-18
+
+### Changed
+
+- dts-scripts: Change to rev removing separate EC update
+- dts-distro.conf: bump to 2.0.0-rc3
+
+## [2.0.0-rc2] - 2024-07-18
+
+### Added
+
+- dts-base-image: add lshw
+
+### Changed
+
+- openssh_%.bbappend: use sshd.service instead of sshd.socket
+- workflow: Change action and jobs names
+- dts-base-image.inc: Remove 'rootfs' suffix from image
+- dts-scripts: Update revision
+- dts-scripts: Add python3-semver dependency
+- meta-dts-distro/recipes-support/dasharo-ectool: bump rev for Dasharo ACPI ID
+- dts-distro.conf: bump version to 2.0.0-rc2
+- dts-scripts: Change to rev removing separate EC update
+- dasharo-ectool: automatically get PV hash from SRCCREV
+- Revert "dts-scripts: Change to rev removing separate EC update"
+- tests: cukinia: Add semver module check
+
+## [2.0.0-rc1] - 2024-07-09
+
+### Added
+
+- distro: support: hidapi: delete
+- dts-distro.conf: add usrmerge to DISTRO_FEATURES
+- dasharo-ectoo: add sha256sums for cargo deps
+
+### Changed
+
+- kas-uefi-sb: change meta-secure-core origin
+- dts: dts-functions: make dts not use ICMP for network connection check
+- distro: recipes-dts: move DTS scripts to another repo
+- distro: recipes-dts: dts-scripts: do_install via Makefile
+- dts-scripts_git.bb: relicense to Apache-2.0
+- distro: dts-scripts: bump revision
+- distro: kernel: acpi-call-dkms: change license to GPLv3
+- update layers to scarthgap
```

Comment:
Consequence of updating layers was also that most of recipes also were updated. Most important of those was probably kernel version change. I think it should be mentioned, somewhere near top.

Reply:
https://github.com/Dasharo/meta-dts/compare/cc1e0b3e1ab0b2b0923e8138d993eac4ed4a9dc8..cd655fffadd262c8ce28b76c20b669d3c9499d14